src/components/organisms/Settings/ProjectList/hooks.ts

```diff
@@ -83,7 +95,24 @@ export default (teamId: string) => {
         : undefined,
     )
     .filter((project): project is Project => !!project && project?.isArchived === true);
+  const hasMoreProjects =
+    projectData?.projects.pageInfo?.hasNextPage || projectData?.projects.pageInfo?.hasPreviousPage;
 
+  const isRefetchingProjects = networkStatus === 3;
+  const getMoreProjects = useCallback(() => {
+    if (hasMoreProjects) {
+      fetchMore({
+        variables: {
+          after: projectData?.projects.pageInfo?.endCursor,
+          first: projectPerPage,
+        },
+        updateQuery: (prev, { fetchMoreResult }) => {
+          if (!fetchMoreResult) return prev;
+          return fetchMoreResult;
+        },
```

Comment:
Same as dashboard, is this necessary even though we are merging in `gql/provider`?

Reply:
as mentioned before , with me I couldn't get ride of it , query fetch only last result

Reply:
as mentioned before , with me I couldn't get ride of it , query fetch only last result

Reply:
same as before , without update , no merging happened

Reply:
This isn't working on my local anyways. It just loads and loads and loads. Also, when I switch to the archived tab with 0 archived projects it shows the loading animation, and if I go back to the normal projects it becomes the stuck loading animation and the projects have disappeared